components/chart/description/index.tsx

```diff
@@ -156,6 +162,7 @@ const Drawer: React.FC<PropsWithChildren<DrawerProps>> = ({ open, onClose, modul
                             </div>
                         </div>
                         <h5 className='sub_title'>{module?.city} {module?.sub_title}</h5>
+                        <h3 className='tags'>{module?.tags}</h3>
```

Comment:
Can we design the tags with colors as next:
https://mui.com/material-ui/react-chip/#color-chip

With spaces between them, no need for commas

Can you please also make the title as h1, and keep the same style we have now